plugins/experimental/maxmind_acl/mmdb.cc

```diff
@@ -406,6 +406,15 @@ Acl::eval(TSRemapRequestInfo *rri, TSHttpTxn txnp)
 {
   bool ret = default_allow;
   int mmdb_error;
+
+  auto sockaddr = TSHttpTxnClientAddrGet(txnp);
+
+  if (sockaddr == nullptr) {
+    TSDebug(PLUGIN_NAME, "Err during TsHttpClientAddrGet, nullptr returned");
+    ret = false;
+    return ret;
+  }
+
```

Comment:
I think you meant to change the next line to use sockaddr instead of calling TSHttpTxnClientAddrGet() again.

Reply:
ahh, yes, thanks, was a quick copy pasta over from 8x